operations/onboarding.md

```diff
@@ -0,0 +1,99 @@
+
+### Preparedness and communicating availability
+
+TOC members, in accordance with the Charter Section 6(f)iii, are expected to attend and be prepared in-advance of each meeting so that the conversation may be productive and decisions may be reached in a timely manner.  TOC members are **not full-time employees of the CNCF**, they are individuals elected to serve the community - many with full-time jobs for which their employers support their activites - *for the benefit of cloud native and open source*.  As such, we understand that incidents, pressing issues, and other primary work priorities may occur from time-to-time, particularly as many TOC members hold significant technical leadership roles within their organizations, that may inhibit their ability to make every meeting all the time. Further, we are human with personal lives and responsibilities that may temporarily limit our ability to attend and participate in these meetings, e.g. vacations, personal matters, etc. It is expected that every TOC member *will communicate their absence or limited availability* in advance (when possible), but endevour to be present and prepared for these meetings. 
+
+In addition to these meetings, TOC members are expected to meet synchronously or asynchronously with the TAGs for which they liaison, occassionally attending TAG meetings.
+
+### Inactivity of a TOC member
+
+If, for any reason, a TOC member is continuously absent without notice, see 6(f)iii, their voting privileges are *suspended* until such time as their consecutive attendence at two meetings. Due to the nature of our work, the inability of a TOC member to cast an *informed* vote (as occurs by meeting attendence *and* involvement in regular engagement as detailed above) penalizes the entire TOC's ability to regularly achieve quorum.  If after the 4th absence the TOC is not notified, the TOC Chair with support of CNCF Staff will make an attempt to address the absence with the TOC member.  If the Chair and Staff are unsuccessful, the Chair will bring the issue of absence to TOC vote in accordance with z6(f)ii at the next private meeting and provide notice to the Governing Board of the outcome. 
+
+If, for whatever reason, a TOC member feels that cannot sustain the committment required of the role, they may proactively notify the TOC Chair or CNCF Staff of their decision to step down.  While uncommon, this is entirely reasonable to have happen, particularly as lives and careers grow and shift. We thank each TOC member for their time of service to the community, however long or brief it may be.
```

Comment:
```suggestion
If, for whatever reason, a TOC member feels they cannot sustain the commitment required of the role, they may proactively notify the TOC Chair or CNCF Staff of their decision to step down. While uncommon, this is entirely reasonable to have to happen, particularly as lives and careers grow and shift. We thank each TOC member for their time of service to the community, however long or brief it may be.
```